### ISSUE TITLE
unphysical structure for villin

### ISSUE THREAD

**Issue by hnadeem2** [kiharalab/Distance-AF#4]
Hello,
I was trying to run Dist-AF for villin headpiece. I got the embeddings from AF2 and then trained Dist-AF, but even after 10k epochs, the structure is nonphysical with numerous erroneous bonds and interactions. I have attached the following files for reproducibility:
1. Dist restraints file: `dist_rest_0_round0.csv`
2. Embeddings file (from AF2 notebook): `model_1.npz`
3. Initial structure (from AF2 notebook) : `model_1.pdb`
4. FASTA file: `villin.fasta`

I use the following command:
`python main.py --target_file=new_Example/villin --emd_file=new_Example/model_1.npz --dist_info=new_Example/dist_rest_0_round0.csv --initial_pdb=new_Example/model_1.pdb --fasta_file=new_Example/villin.fasta --output_dir=./new_example_output --model_dir=./model_dir --dist_weight=0.5 --loose_dist=1 --epochs=10000 --device_id=0`

[new_Example.zip](https://github.com/kiharalab/Distance-AF/files/15408315/new_Example.zip)
![dist](https://github.com/kiharalab/Distance-AF/assets/98351113/ecfc20f6-25b4-4eae-90ca-c44ec7a15271)

**Comment by Zhang038** [comment]
Hello, thank you for your interest in our work. I reviewed the file you uploaded and noticed the following points:

1. You are missing an important file called --window_info. This file defines the separate domains for which you want to change the relative orientation. You can find an example [here](https://github.com/kiharalab/Distance-AF/blob/main/Example/1IXCA/window.txt), with a detailed explanation in the README file.
2. Additionally, the distance constraint file you provided is pair-wise, while dist-af expects constraints defined across separate domains. It seems you are aiming for local structure refinement rather than a significant change in domain orientation. When dist-af receives constraints within the same domain, it often causes local structure clashes due to the loss function definition.
3. For better performance, consider adjusting your input files to be more compatible with dist-af's settings. Define the domain regions clearly and reduce the number of constraints, keeping only those that span across domains.

Please let me know if any further questions.

**Comment by hnadeem2** [comment]
Hi,
Thanks for your reply.

If I want structure refinement inside a single domain (like the example I posted above), would it be possible using Distance AF?

**Comment by Zhang038** [comment]
Hi, I would assume it will not work well with default settings, sorry for that. As distance-af is designed for domain based orientation movement.

**Comment by hnadeem2** [comment]
Understood. Thank you for your feedback.
